chain/yoda/execute.go

```diff
@@ -50,17 +53,30 @@ func SubmitReport(c *Context, l *Logger, id otypes.RequestID, reps []otypes.RawR
 		l.Error(":exploding_head: Failed to build tx with error: %s", err.Error())
 		return
 	}
-
-	res, err := cliCtx.BroadcastTxCommit(out)
-	if err != nil {
+	var res sdk.TxResponse
+	for try := 1; try <= MaxTry; try++ {
+		l.Info("Try to broadcast: %d/%d", try, MaxTry)
+		res, err = cliCtx.BroadcastTxSync(out)
+		if err == nil {
+			break
+		}
 		l.Error(":exploding_head: Failed to broadcast tx with error: %s", err.Error())
-		return
+		time.Sleep(SleepTime)
 	}
-	if res.Code != 0 {
-		l.Error(":exploding_head: Tx returned nonzero code %d with log %s, tx hash: %s", res.Code, res.RawLog, res.TxHash)
+	for start := time.Now(); time.Since(start) < c.broadcastTimeout; {
```

Comment:
What happens after trying until MaxTry and cannot send tx (got error every try) We need to query on empty tx response?